en/docs/guides/identity-federation/configure-saml-2.0-web-sso.md

```diff
@@ -21,26 +21,24 @@ directly accessing the identity provider itself.
 
 Follow the steps below to navigate to the federated authenticators configuration section.
 
-!!! tip "Before you begin"
 
-	1.	[Run WSO2 Identity Server](../../../deploy/get-started/run-the-product/).
 
-	2.	Sign in to the WSO2 Identity Server [Management Console](../../../deploy/get-started/get-started-with-the-management-console/) at `https://<SERVER_HOST>:9443/carbon` as an administrator. 	  
+1.	Sign in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) as an administrator. 	  
 
    
-1.  On the **Main** menu of the Management Console, click **Identity > Identity Providers > Add**.
+2.  On the **Main** menu of the Management Console, click **Identity > Identity Providers > Add**.
 
 	<img src="../../../assets/img/guides/add-identity-provider-menu-item.png" alt="Add Identity Provider menu-item" width="200" style="border:1px solid grey">  
     
-2.  Enter a suitable name for the identity provider in the **Identity Provider Name** text box.
+3.  Enter a suitable name for the identity provider in the **Identity Provider Name** text box.
 
 	<img src="../../../assets/img/guides/add-identity-provider-screen.png" alt="Add Identity Provider screen" width="700" style="border:1px solid grey"> 
```

Comment:
instead of this image can't we reuse `assets/img/fragments/add-new-identity-provider-screen.png` ?